src/firmware/app/control/control.h

```diff
@@ -16,14 +16,16 @@ void app_control_applyAccel(const FirmwareRobot_t* robot, float linear_accel_x,
                             float linear_accel_y, float angular_accel);
 
 /**
- * Drive the given robot at the given velocity in the global coordinate frame
+ * Drive the given robot at the given velocity in the robot coordinate frame
  *
  * @param robot The robot to move at the given velocity
- * @param linear_velocity_x The velocity to move at in the x-direction in the global
+ * @param linear_velocity_x The velocity to move at in the x-direction in the robot
  *                          coordinate frame (m/s)
- * @param linear_velocity_y The velocity to move at in the y-direction in the global
+ * @param linear_velocity_y The velocity to move at in the y-direction in the robot
  *                          coordinate frame (m/s)
  * @param angular_velocity The angular velocity to move at (rad/s)
  */
-void app_control_trackVelocity(FirmwareRobot_t* robot, float linear_velocity_x,
-                               float linear_velocity_y, float angular_velocity);
+void app_control_trackVelocityInLocalFrame(FirmwareRobot_t* robot,
```

Comment:
Not a blocking comment, but we seem to say `robot frame` for the robots frame of reference and `global frame` for the frame of reference using the fields coordinate system?

For sake of consistency should this be `trackVelocityInRobotFrame`? Not blocking, just a minor comment

Reply:
This is blocking for me

Reply:
Done.